resources/settings.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <settings>
     <setting id="last_update" value="0.0" visible="false"/>
+    <setting id="last_update_check" value="0.0" visible="false"/>
```

Comment:
I think it is weird that these are marked as floats, but I guess the timestamp returns sub-second timestamps?

Reply:
I didn't really think about that, but yes, mktime returns a float.

Reply:
Just had another look. The timetuple doesn't have ms so it should always be .0 and it could be saved as int. But there is no real need for that.

Reply:
I added a commit which (imho) simplifies the code a little. With this last_update and last_check are "real" floats.